docs/comunidad/charlas-y-presentaciones.md

```diff
@@ -0,0 +1,122 @@
+# **Presentaciones y vídeos**
+
+En esta sección encontrarás presentaciones y vídeos de charlas impartidas por nosotros.
+
+___
+
+![EOS Costa Rica pre practica](/static/img/charlas/Pre-practica.webp)
```

Comment:
- Fix image paths (remove `static`)

here is an example but you will need to do this for the rest of the images

![EOS Costa Rica pre practica](/img/charlas/Pre-practica.webp)